package.json

```diff
@@ -98,5 +101,6 @@
     "exclude": [
       "src/serviceWorker.js"
     ]
-  }
+  },
+  "packageManager": "yarn@3.0.1-rc.1"
```

Comment:
Does this mean that every time we want to upgrade the package manager we need to edit this line and change the file in `.yarn/releases`? Or is it done automatically?

Reply:
Everything is automatic!